pyproject.toml

```diff
@@ -19,33 +19,43 @@ classifiers = [
 ]
 
 dependencies = [
-    "fastapi>=0.100.0",
-    "sqlalchemy>=2.0.19",
-    "uvicorn>=0.23.0",
-    "pydantic[dotenv]<2.0.0",
+    "fastapi[all]>=0.109.0",
+    "sqlalchemy>=2.0.25",
+    "uvicorn>=0.27.0",
+    "pydantic>=2.6.0",
+    "pydantic-settings>=2.1.0",
     "python-jose>=3.3.0",
-    "passlib>=1.7.4",
-    "bcrypt>=3.2.0",
-    "numpy~=1.24.4",
-    "pandas~=1.5.3",
-    "openpyxl>=3.0.9",
-    "FINE>=2.2.2",
-    "python-multipart>=0.0.6",
+    "passlib[bcrypt]>=1.7.4",
+    # dependencies for FINE:
```

Comment:
why add dependencies of fine? they are already specified as dependencies.
I would not like to keep track of this?

Reply:
FINE moved their dependencies to conda-forge. Installing FINE from pip no longer installs their dependencies. So either we also move to conda-forge or list the dependencies for FINE to install from pip ourselves.

Reply:
I see.
That is not good at all..
So the problem is here:
https://github.com/FZJ-IEK3-VSA/FINE/blob/master/pyproject.toml#L32

I think, I would rather file a PR to FINE to fix this..?